modules/ocf_desktop/manifests/packages.pp

```diff
@@ -53,12 +53,15 @@
 
   # Remove some packages
   package {
+    # causes the fcitx menu to vanish
```

Comment:
Maybe say what the fcitx menu does for those who don't know it is for switching keyboard layouts?